cmd/snap/cmd_run.go

```diff
@@ -762,6 +762,31 @@ func snapdHelperPath(toolName string) (string, error) {
 	return filepath.Join(snapBase, dirs.CoreLibExecDir, toolName), nil
 }
 
+/* Kerberos tickets live in /tmp, or in the place specified by KRB5CCNAME
+ * environment variable. However, in snaps /tmp is private.
+ * So rewire the environment variable to point to /var/lib/snapd/hostfs/tmp,
+ * in case the variable is unset or resolves tickets to /tmp/krb5cc*.
+ *
+ * Snaps that want to read Kerberos tickets must then connect to the
+ * kerberos-tickets interface to have access to the tickets.
+ */
+func exposeKerberosTickets(info *snap.Info) (string, error) {
+	krb5EnvVar := osGetenv("KRB5CCNAME")
+	if len(krb5EnvVar) == 0 {
+		return "", nil
+	}
+	if strings.HasPrefix(krb5EnvVar, "FILE:") {
+		path := strings.TrimPrefix(krb5EnvVar, "FILE:")
+		path = filepath.Clean(path)
+		if filepath.Dir(path) == "/tmp" && strings.HasPrefix(path, "/tmp/krb5cc") {
```

Comment:
The check is redundant, please keep only the `HasPrefix` part.

Reply:
It's not because without the first check `/tmp/krb5cc/a/b/c` would pass the test.